### PR TITLE
adds a greyscale coat with a concerningly high number of colorable sections

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
Adds a single coat, but this coat as a powerful 9 separate colorable sections for making a massive variety of designs from.

## How This Contributes To The Skyrat Roleplay Experience
Cool greyscale items are good for the skyrat roleplay experience aren't they

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
Just a few examples of what you can make
  
image

image

image

image

</details>

## Changelog

:cl:
add: A new recolorable coat with a large number of customization options has been added
/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
the sprite maintainer fears the clothing item with nine color layers

**Review by Ebin-Halcyon — APPROVED** [review]
I mean, the sprites are fine, but jesus. 9 layers?

**Review by tf-4 — APPROVED** [review]
9 layers...

**Comment by ghost** [comment]
Mind showing some side views? I'm

Intrigued

File: modular_skyrat/modules/GAGS/json_configs/suits/tailored_jacket/tailored_jacket.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+			"color_ids": [ 8 ]
+		},
+		{
+			"type": "icon_state",
+			"icon_state": "longcoat_armstripes_long",
+			"blend_mode": "overlay",
+			"color_ids": [ 9 ]
+		}
+	],
+	"tailored_jacket_t": [
```
Comment (ghost) [inline modular_skyrat/modules/GAGS/json_configs/suits/tailored_jacket/tailored_jacket.json:58]: oh my god it's even toggleable

File: modular_skyrat/modules/GAGS/json_configs/suits/tailored_jacket/tailored_jacket.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
+{
+	"tailored_jacket": [
+		{
+			"type": "icon_state",
+			"icon_state": "longcoat_base_obj",
```
Comment (ghost) [inline modular_skyrat/modules/GAGS/json_configs/suits/tailored_jacket/tailored_jacket.json:5]: Out of curiosity is there a reason for all the component states being longcoat with the full icon being tailored_jacket? I'd hate to say you **need** to keep them consistent so don't worry, just something to keep in mind in future if anything

Comment (Paxilmaniac) [reply modular_skyrat/modules/GAGS/json_configs/suits/tailored_jacket/tailored_jacket.json:5]: the answer is i didn’t know what i was going to call this thing at this point

**Comment by Paxilmaniac** [comment]
> Mind showing some side views? I'm
> 
> Intrigued

gotchu homie

image

image

image

File: modular_skyrat/modules/GAGS/json_configs/suits/tailored_jacket/tailored_jacket.json (hunk 1 above)
Comment (Paxilmaniac) [reply modular_skyrat/modules/GAGS/json_configs/suits/tailored_jacket/tailored_jacket.json:58]: 'toggleable' means rolling up the sleeves and not much more, however

**Pull request merged**